scout/adapter/mongo/case.py

```diff
@@ -230,9 +229,7 @@ def _populate_name_query(self, query, name_query, owner=None, collaborator=None)
         if query_field == "user":
             query_terms = query_term.split(" ")
             user_query = {
-                "$and": [
-                    {"name": {"$regex": re.escape(term), "$options": "i"}} for term in query_terms
-                ]
+                "$and": [{"name": {"$regex": term, "$options": "i"}} for term in query_terms]
```

Comment:
Removing this one is likely overkill, but we can go through it more calmly another day. The patch works.